core/src/main/scala/org/apache/spark/network/netty/SparkTransportConf.scala

```diff
@@ -55,4 +58,32 @@ object SparkTransportConf {
       }
     })
   }
+
+  /**
+   * Separate threads configuration of driver and executor
+   * @param conf the [[SparkConf]]
+   * @param module the module name
+   * @param server if true, it's for the serverThreads. Otherwise, it's for the clientThreads.
+   * @param defaultNumThreads default number of threads
+   * @return
+   */
+  def getNumOfThreads(
+      conf: SparkConf,
+      module: String,
+      server: Boolean,
+      defaultNumThreads: Int): String = {
```

Comment:
:nit Can be: Int, cast it to String outside this function

Reply:
I put the cast inside this function. what's the difference or benefit if I put it outside of the function?
thanks.

Reply:
numThreads is more natural to be a Int